packages/apm/src/hubextensions.ts

```diff
@@ -45,7 +45,10 @@ function startSpan(spanOrSpanContext?: Span | SpanContext, forceNoChild: boolean
   if (!isSpanInstance(spanOrSpanContext) && !forceNoChild) {
     if (scope) {
       const parentSpan = scope.getSpan() as Span;
-      if (parentSpan) {
+      // If we have a span on the scope, it means we want to create a child on this span
+      // but only if there is no timestamp set. This means that this span has already be finished.
```

Comment:
```suggestion
      // but only if there is no timestamp set. This means that this span has already finished.
```

This is a confusing expectation/requirement :cry: 

What is the case that you encountered that motivated this change?